Fix - Update SDK docs (#1047)
iotaledger/iota-wiki: next/docs/iota-sdk/docs/how-tos/accounts-and-addresses/list-accounts.mdx

import TabItem from '@theme/TabItem';
import TipAccountApproaches from '../../_admonitions/_tip-account-approaches.md';

> As your application probably will handle many accounts or addresses, the IOTA SDK provides a convenient function to list
> all the [`Accounts`](../../../references/nodejs/classes/Account/) related to a wallet,

Reviewer: ```suggestion
As your application may handle many accounts, the IOTA SDK provides a convenient function to list
all of the [`Accounts`](../../../references/nodejs/classes/Account/) related to a wallet,
```
